apps/web/src/components/settings/account-settings.tsx

```
@@ -0,0 +1,263 @@
+      </div>
+
+      {/* Connected accounts */}
+      <div>
+        <h3 class="mb-4 text-sm font-semibold uppercase tracking-wider text-muted-foreground">
+          Connected Accounts
+        </h3>
+        <div class="space-y-2">
+          <div class="flex items-center justify-between rounded-lg border border-border bg-card p-3">
+            <div class="flex items-center gap-3">
+              <div class="flex h-8 w-8 items-center justify-center rounded-full bg-[#5865F2]">
+                <svg class="h-4 w-4 text-white" viewBox="0 0 24 24" fill="currentColor">
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)